internal/js/jsmodules.go

```diff
@@ -32,43 +30,48 @@ import (
 	return map[string]interface{}{
-		"k6":                      k6.New(),
-		"k6/crypto":               crypto.New(),
-		"k6/crypto/x509":          x509.New(),
-		"k6/data":                 data.New(),
-		"k6/encoding":             encoding.New(),
-		"k6/timers":               timers.New(),
-		"k6/execution":            execution.New(),
-		"k6/experimental/csv":     csv.New(),
-		"k6/experimental/redis":   redis.New(),
-		"k6/experimental/streams": streams.New(),
-		"k6/experimental/webcrypto": newWarnExperimentalModule(webcrypto.New(),
-			"k6/experimental/webcrypto is now part of the k6 core, and globally available. You could just remove import."+
-				" The k6/experimental/webcrypto will be removed in k6 v1.1.0"),
+		// Stable modules
+		"k6":             k6.New(),
+		"k6/browser":     browser.New(),
+		"k6/crypto":      crypto.New(),
+		"k6/crypto/x509": x509.New(),
+		"k6/data":        data.New(),
+		"k6/encoding":    encoding.New(),
+		"k6/execution":   execution.New(),
+		"k6/html":        html.New(),
+		"k6/http":        http.New(),
+		"k6/net/grpc":    grpc.New(),
+		"k6/metrics":     metrics.New(),
+		"k6/secrets":     secrets.New(),
+		"k6/timers":      timers.New(),
+		"k6/ws":          ws.New(),
+
+		// Experimental modules
+		"k6/experimental/csv":        csv.New(),
+		"k6/experimental/fs":         fs.New(),
+		"k6/experimental/redis":      redis.New(),
+		"k6/experimental/streams":    streams.New(),
 		"k6/experimental/websockets": expws.New(),
-		"k6/experimental/timers": newRemovedModule(
-			"k6/experimental/timers has been graduated, please use k6/timers instead."),
-		"k6/experimental/tracing": newRemovedModule(
-			"k6/experimental/tracing has been removed. All of it functionality is available as pure javascript module." +
-				" More info available at the docs:" +
-				" https://grafana.com/docs/k6/latest/javascript-api/jslib/http-instrumentation-tempo"),
+
+		// Removed modules
 		"k6/experimental/browser": newRemovedModule(
 			"k6/experimental/browser has been graduated, please use k6/browser instead." +
 				"Please update your imports to use k6/browser instead of k6/experimental/browser," +
 				" For more information, see the migration guide at the link:" +
 				" https://grafana.com/docs/k6/latest/using-k6-browser/migrating-to-k6-v0-52/"),
-		"k6/browser":         browser.New(),
-		"k6/experimental/fs": fs.New(),
-		"k6/net/grpc":        grpc.New(),
-		"k6/html":            html.New(),
-		"k6/http":            http.New(),
-		"k6/metrics":         metrics.New(),
-		"k6/secrets":         secrets.New(),
-		"k6/ws":              ws.New(),
 		"k6/experimental/grpc": newRemovedModule(
 			"k6/experimental/grpc has been graduated, please use k6/net/grpc instead." +
 				" See https://grafana.com/docs/k6/latest/javascript-api/k6-net-grpc/ for more information.",
 		),
+		"k6/experimental/timers": newRemovedModule(
+			"k6/experimental/timers has been graduated, please use k6/timers instead."),
```

Comment:
Is this perhaps an opportunity to slightly modify this message, and tell people that they're even available globally?